recipes/mvapich2/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = "3.0" %}
+
+package:
+  name: mvapich
+  version: {{ version }}
+
+source:
+  url: https://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich-{{ version }}.tar.gz
+  sha256: ee076c4e672d18d6bf8dd2250e4a91fa96aac1db2c788e4572b5513d86936efb
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  run_exports:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+
+requirements:
+  build:
+    - bison
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib("c") }}
+    - make
+  host:
+    - libhwloc
+    - rdma-core  # [linux]
+  run:
+    - mpi 1.0 mvapich
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
```

I don't imagine this mpi implementation needs all the compilers to run, does it?

```suggestion
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - {{ compiler('fortran') }}
```

---

OK